### DISCUSSION TITLE
Allow default story to be infered from default export in CSF3

### DISCUSSION THREAD

**Discussion by unrevised6419** [storybookjs/storybook#21229] [Ideas]
In most of the projects, our components/sections have a single story, so most of our stories looked like this in CSF2:

<details>
<summary>CSF2 Example with createDefaultStory</summary>

**accordion.stories.ts**
```ts
import { Meta } from '@storybook/vue3';
import Component, { AccordionProps as ComponentProps } from './accordion.component.vue';
import { createDefaultStory } from '~/utils/storybook';

export default {
	title: 'Components / Accordion',
	component: Component,
	args: { /* ... */ },
} as Meta<ComponentProps>;

export const Accordion = createDefaultStory(Component);
```

**utils/storybook.ts**
```ts
import { Args, StoryFn } from '@storybook/vue3';
import { Component, defineComponent } from 'vue';

export function createDefaultStory<TArgs = Args>(
	Block: Component,
): StoryFn<TArgs> {
	return (args: TArgs) => {
		return defineComponent({
			components: { Block },
			setup: () => ({ args }),
			template: `<Block v-bind="args" />`,
		});
	};
}
```

</details>

With the arrival of CSF3, we don't need anymore this `createDefaultStory` utility function that creates the story, we can use a simple object named export:

**accordion.stories.ts**
```ts
import { Meta, StoryObj } from '@storybook/vue3';
import Component, { AccordionProps as ComponentProps } from './accordion.component.vue';

export default {
	title: 'Components / Accordion',
	component: Component,
	args: { /* ... */ },
} as Meta<ComponentProps>;

export const Accordion: StoryObj = {};
```

Would be nice if there are no stories exported from the file, Storybook to automatically infer a default story from the `component` and `title` fields, so we could get rid completely of the named export

```ts
export const Accordion: StoryObj = {};
```

At the moment removing this single named export `StoryObj`, eslint will complain, and no stories will be detected by Storybook

**Comment by francoisfaubert** [comment]
I think this request makes even more sense when you have `autodocs` enabled.

The component view in the generated docs page will likely be the same as the one exported by the empty `StoryObj`.
